blockdag/error.go

```diff
@@ -217,6 +217,10 @@ const (
 	// ErrInvalidParentsRelation indicates that one of the parents of a block
 	// is also an ancestor of another parent
 	ErrInvalidParentsRelation
+
+	// ErrDelayedBlockisNotAllowed indicates that a block with a delayed timestamp was
+	// submitted and should be rejected.
```

Comment:
instead of saying "and should be rejected" maybe write "in a context that it shouldn't be allowed", or simply write "with BFDisallowDelay flag raised" or something like this?